Content.Client/Players/PlayTimeTracking/JobRequirementsManager.cs

```diff
@@ -177,4 +176,24 @@ public TimeSpan FetchOverallPlaytime()
 
         return _roles;
     }
+
+    public bool IsRoleBanned(IEnumerable<string> roles, out string? banReason, out DateTime? expirationTime)
+    {
```

Comment:
You should just use NotNullWhen(true) and then you can remove the null checks in the calls above.